controllers/homeRoutes.js

```diff
@@ -23,12 +27,6 @@ router.get('/goals/:goal_id', async (req, res) => {
   }
 });
 
-
-
-
-
-module.exports = router;
```

Comment:
Did you delete module.exports = router entirely from this route? Should definitely stay in there.

Reply:
My apologies